core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiServiceSpec.scala

```diff
@@ -2801,7 +2870,13 @@ class EntityApiServiceSpec extends ApiServiceSpec {
       }
   }
 
-  class PaginationTestData extends TestData {
+  // TODO CORE-637 quicksilver rounding is off
+  def getRandomWithPrecision(digits: Int): Double = {
+    val factor = Math.pow(10, digits)
+    Math.round(Math.random() * factor) / factor
+  }
```

Comment:
hooboy I think this is an interesting one, I wonder if this is actually a difference in MySQL behavior about number precision nested in a json column vs. number precision in a standalone numeric column. I thought both were doubles. https://stackoverflow.com/questions/78530131/mysql-loss-precision-while-converting-json-string-to-double is an interesting read.